focus-blocked/index.html

```diff
@@ -108,8 +108,8 @@ <h1 id="focusTitle"></h1>
       const focusEndTime = urlParams.get("focus_mode_end_time");
       const block_type = urlParams.get("block_type")
 
-      if(block_type === "always-block") {
-        document.getElementById("focusTitle").innerText ="You said this URL should always be blocked";
+      if(block_type === "always-blocked") {
```

Comment:
can you do `.includes('always-blocked')`? Maybe windows handles it differently from Mac